vos/vos/vos.py

```diff
@@ -1844,8 +1868,9 @@ def copy(self, source, destination, send_md5=False, disposition=False,
                     put_urls = self.get_node_url(destination, 'PUT',
                                                  content_length=src_size,
                                                  md5_checksum=dest_md5)
+                    negotiated = False
                     while not success:
-                        if len(put_urls) == 0:
+                        if len(put_urls) == 0 and not negotiated:
```

Comment:
Does this same check need to be added to line 1921, which is the second `while not success` loop?

Reply:
That flag was not necessary as there was another one serving the same purpose (but only better) so I've removed from everywhere.